src/styles/components/button.scss

```diff
@@ -0,0 +1,24 @@
+.button {
+    outline: 0;
+    border: none;
+    background-color: white;
+    color: steelblue;
+    font-weight: 600;
+    height: 1.3rem;
+    padding: 0 0.5rem;
+    transition: 0.3s ease;
```

Comment:
to get `transition` work, there should be listed things to transition. in this case:
```
transition: background-color 0.3s ease, color 0.3.s ease;
```
also, there is a possibility to write like:
`transition: all 0.3s ease;`